project-base/storefront/tailwind.config.js

```diff
@@ -14,36 +14,28 @@ module.exports = {
             xxl: em(1560),
```

Comment:
Arrow in banners slider buttons is not visible

https://github.com/shopsys/shopsys/assets/55181354/bbfe6f6b-5eb7-4d34-996e-f70df2dec051